app/src/main/res/layout/fr_logic_editor.xml

```diff
@@ -7,6 +7,10 @@
             <com.besome.sketch.editor.view.ViewLogicEditor android:id="@id/editor" android:layout_width="fill_parent" android:layout_height="fill_parent" android:layout_margin="4.0dip" />
         </LinearLayout>
     </RelativeLayout>
-    <com.besome.sketch.editor.view.ViewDummy android:id="@id/dummy" android:layout_width="fill_parent" android:layout_height="fill_parent" />
+
+    <com.besome.sketch.editor.view.ViewDummy
+        android:id="@id/dummy"
+        android:layout_width="fill_parent"
+        android:layout_height="fill_parent" />
```

Comment:
These changes don't fit to the About Sketchware Pro activity changes. Please at least mention or create a separate Pull request for them next time.